common/djangoapps/third_party_auth/models.py

```diff
@@ -244,6 +261,12 @@ class OAuth2ProviderConfig(ProviderConfig):
             # To be precise, it's set by AUTHENTICATION_BACKENDS - which aws.py sets from THIRD_PARTY_AUTH_BACKENDS
         )
     )
+    provider_slug = models.SlugField(
```

Comment:
The `LTIProviderConfig` identifies itself using the `lti_consumer_key`, which is a 255 character string. I think it makes sense to key LTI this way and not with a separate slug, so I feel that abstracting this into the common `ProviderConfig` might not be the right solution. Thoughts?